Annotations/PdfViewerAnnotations/PdfViewerAnnotations/MainPage.xaml.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Syncfusion.Maui.PdfViewer;
+using System.Collections.ObjectModel;
+using System.Reflection;
+
+namespace PdfViewerAnnotations
+{
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+            Stream? stream = typeof(MainPage).GetTypeInfo().Assembly.GetManifestResourceStream("PdfViewerAnnotations.Assets.Annotations.pdf");
+            PDFViewer.LoadDocument(stream);
+        }
+
+        private void AddHighlightAnnotation(object sender, EventArgs e)
+        {
+            // Create a highlight annotation.
+            HighlightAnnotation highlightAnnotation = CreateHighlightAnnotation();
+
+            // Add the highlight annotation to the PDF document using the AddAnnotation method of the `SfPdfViewer` instance.
+            PDFViewer.AddAnnotation(highlightAnnotation);
+        }
+
+        HighlightAnnotation CreateHighlightAnnotation()
+        {
+            int pageNumber = 1;
+
+            // Create a list of text bounds that represents a multiple lines of text markups. Here a single line text markup are created.
+            List<RectF> textBoundsCollection = new List<RectF> { new RectF(70, 70, 260, 40) };
+
+            // Create an highlight annotation.
+            HighlightAnnotation annotation = new HighlightAnnotation(pageNumber, textBoundsCollection);
+
+            // Set the appearance for the highlight annotation
+            annotation.Color = Colors.BlueViolet; // set the highlight color
+            annotation.Opacity = 0.5f; // set the opacity to 50%
+
+            return annotation;
+        }
+
+        private void RemoveAnnotation(object sender, EventArgs e)
+        {
+            //Obtain the annotation collection.
+            ReadOnlyObservableCollection<Annotation> annotations = PDFViewer.Annotations;
+
+            //Obtain the first annotation in the annotation collection.
+            Annotation firstAnnotation = annotations[3];
```

Comment:
1. The variable name is first annotation but it looks accessing annotation from the 3rd index. Can you please check?
2. Also before directly accessing annotation from index, please add a condition check to avoid exceptions